bob-backend/src/pearl/disk_controller.rs

```diff
@@ -616,18 +625,29 @@ impl DiskController {
         }
     }
 
-    pub(crate) async fn delete_alien(&self, op: Operation, key: BobKey) -> Result<u64, Error> {
+    pub(crate) async fn delete_alien(
+        &self,
+        op: Operation,
+        key: BobKey,
+        force_delete: bool,
+    ) -> Result<u64, Error> {
         if *self.state.read().await == GroupsState::Ready {
-            let vdisk_group = self.find_group(&op).await;
-            if let Ok(group) = vdisk_group {
-                group.delete(key).await
-            } else {
-                debug!(
-                    "DELETE[alien][{}] No alien group has been created for vdisk #{}",
-                    key,
-                    op.vdisk_id()
-                );
-                Err(Error::key_not_found(key))
+            let vdisk_group = self.get_or_create_pearl(&op).await;
```

Comment:
`force_delete` should affect the behavior: if it is `false` then we don't need to create pearl (we need only get existing)